### ISSUE TITLE
Crash  by the accumulation of aliens

### ISSUE THREAD

**Issue by qq578023708** [ReactiveDrop/reactivedrop_public_src#507]
When the onslaught is enabled, if the Marine is located in a place where aliens can't reach, such as sentry gun the fort, or after being knocked unconscious, the game continues to generate a large number of aliens, causing Katon and the server to crash, triggering the error prompt: CUTLRBTree overflow, I can't repair them

```c
if ( m_FirstFree == InvalidIndex() )
	{
		Assert( m_Elements.IsValidIterator( m_LastAlloc ) || m_NumElements == 0 );
		typename M::Iterator_t it = m_Elements.IsValidIterator( m_LastAlloc ) ? m_Elements.Next( m_LastAlloc ) : m_Elements.First();
		if ( !m_Elements.IsValidIterator( it ) )
		{
			MEM_ALLOC_CREDIT_CLASS();
			m_Elements.Grow();

			it = m_Elements.IsValidIterator( m_LastAlloc ) ? m_Elements.Next( m_LastAlloc ) : m_Elements.First();

			Assert( m_Elements.IsValidIterator( it ) );
			if ( !m_Elements.IsValidIterator( it ) )
			{
				Error( "CUtlRBTree overflow!\n" );
			}
		}
		m_LastAlloc = it;
		elem = m_Elements.GetIndex( m_LastAlloc );
		Assert( m_Elements.IsValidIterator( m_LastAlloc ) );
	}
```

**Comment by BenLubar** [comment]
Do you have a mdmp file from when this crash happened?

**Comment by qq578023708** [comment]
Yes, I have a lot of such documents, because this happens too often.
Here are some files I uploaded.
[https://github.com/qq578023708/Reactivedrop_CrashMdmp.git](url)

**Comment by BenLubar** [comment]
`srcds_crash_2023_2_15T3_30_58C0.mdmp` is #382
`srcds_crash_2023_2_16T11_37_27C0.mdmp` looks like memory corruption
`srcds_crash_2023_2_17T1_58_4C0.mdmp` was fixed in a54c3ec26553dc97c1a8ed46d54bd7a92ab44043

**Comment by qq578023708** [comment]
I uploaded 2 new dump files. I don't know if they are the same error. Can I debug these dump files myself? I tried to use visual studio to load them. It told me that the engine.pdb symbol file was missing and whether I had done anything wrong
